source/class/qx/ui/menu/Menu.js

```diff
@@ -506,6 +519,17 @@ qx.Class.define("qx.ui.menu.Menu",
       }
     },
 
+    // property apply
+    _applyOpener : function(value, old)
+    {
+      // ARIA attrs
+      if (value) {
+        this.getContentElement().setAttribute("aria-labelledby", value);
+      } else {
+        this.getContentElement().removeAttribute("aria-labelledby");
```

Comment:
On latest master, I'm getting an error here on dispose because `this.getContentElement()` is null when the call to `setContextMenu(null)` is made from Widget https://github.com/qooxdoo/qooxdoo/blob/28066cf52fe240d009986e67fde9768d4b5ec714/source/class/qx/ui/core/Widget.js#L4004 I've tried to build a repl but can't seem to replicate. Does it just make sense to check the ContentElement here anyway?